xpti/src/xpti_proxy.cpp

```diff
@@ -420,6 +428,10 @@ XPTI_EXPORT_API bool xptiTraceEnabled() {
   return false;
 }
 
+XPTI_EXPORT_API void xptiTraceTryToEnable() {
+  xpti::ProxyLoader::instance().tryToEnable();
```

Comment:
I don't think this changes anything, so xptiTraceTryToEnable() is redundant. Everything is controlled by environment variables.

Reply:
yes, it is controlled by environment variables but no way to enable xpti traces programmatically - set env vars in runtime.

Reply:
@tovinkere let me to share more details about runtime enabling I want to add. XPTI_TRACE_ENABLE allows to enable traces only if other variables were set at the program loading time. There is no way to requery the state of these variables . If they are set programmatically from code - no chance to use xpti at all. I want to add this functionality to be able to do unit tests.
  unittest::ScopedEnvVar PathToXPTIFW{"XPTI_FRAMEWORK_DISPATCHER",
                                      "libxptifw.so", [] {}};
  unittest::ScopedEnvVar XPTISubscriber{"XPTI_SUBSCRIBERS",
                                        "libxptitest_subscriber.so", [] {}};

Reply:
discussed in person and updated impl.